### DISCUSSION TITLE
Weight Painting with the Molecular Nodes not working?

### DISCUSSION THREAD

**Discussion by vagra98** [BradyAJohnston/MolecularNodes#336] [Q&A]
Trying to do some rigging with trajectory imported as .pdb file with various frames but the bones do not recognise the mesh. The only solution I've found is applying the Molecular Node modifier which creates the mesh for a single frame, thus not allowing for rigging of the MD trajectory. Is there any way in which to preserver the rigging along the frames? Thanks in advance, this would be huge for trajectory animation 🥇

**Comment by BradyAJohnston** [comment]
I just had a play around, and I was able to get it working, you just have to move the Armature Modifier to before the MolecularNodes Modifier as below: 
![image](https://github.com/BradyAJohnston/MolecularNodes/assets/36021261/197926ea-7bfd-489a-93fd-28d30404072a)

https://github.com/BradyAJohnston/MolecularNodes/assets/36021261/c85288ba-c711-4ffa-be57-cfe3d2d80f31
